backend/molgenis-emx2-sql-it/src/test/java/org/molgenis/emx2/sql/TestInherits.java

```diff
@@ -71,6 +76,30 @@ public void testExtends() {
       System.out.println("Errored correctly:\n" + e);
     }
 
+    // try to add column in superclass that already exists in any subclass
+    try {
+      person.getMetadata().add(column("salary").setType(DATE));
+      fail("should fail: cannot add column in superclass to name that already exists in subclass");
+    } catch (MolgenisException e) {
+      System.out.println("Errored correctly:\n" + e);
+    }
+
+    // try to rename column in superclass that already exists in any subclass
+    try {
+      person.getMetadata().alterColumn("birthDate", column("salary").setType(DATE));
+      fail("should fail: cannot rename column to superclass that already exists in subclass");
+    } catch (MolgenisException e) {
+      System.out.println("Errored correctly:\n" + e);
+    }
+
+    // try to alter name in superclass that already exists in any subclass IN OTHER SCHEMA
+    try {
+      person.getMetadata().add(column("title").setType(DATE));
+      fail("should fail: cannot add column in superclass to name that already exists in subclass");
+    } catch (MolgenisException e) {
+      System.out.println("Errored correctly:\n" + e);
+    }
+
```

Comment:
same test for sub class ? , test for n level inheritance ?

Reply:
Motivation for second test is that I wanted to test both multi-level AND cross schema inheritance.

Reply:
This is not clear form the test name. In general; create i test for each case and use a descriptive name ( optionally add description ). This not only helps with code review and extending / removing  tests but is show when test fails in CI context

Reply:
The comment above was explanatory if you read the uppercase letters. 

I don't feel to refactor the whole test suite now to be split in all kind of little tests now. Because, until now we have not discussed changing our pattern of test writing with the desire to split in little tests. I would like to be consistent.

I might be tempted to try that with you in a new test suite. However, I feel this will result in code verbosity as well as in runtime overhead (if applied to total test suite, it would 5-10x the set) which is one of the reasons I have not done this so far. But maybe we like it.

Reply:
Hmm, having slept over it I noticed what might be the real issue: the fail message was the same. Updated it.